mt-kahypar/datastructures/static_graph.cpp

```diff
@@ -28,17 +28,16 @@
 
 #include "static_graph.h"
 
+#include <algorithm>
+
+#include "tbb/parallel_reduce.h"
```

Comment:
I think the library includes should stay with <>. Just saw this now. Will do a proper read-through later.

Reply:
nope most of them were included in this style before. Btw, I will not go over all files again. This was the most annoying work I've ever done!!!

Reply:
Well, both are used very inconsistently at the moment. I don't really have a preference, but this PR might be a good opportunity to do it uniformly everywhere

Reply:
Yes, they are uniform now 😊

Reply:
btw. we have to transistion to a `include-what-you-use` policy, otherwise reordering include becomes a real pain.

Reply:
Yeah, `include-what-you-use` definitively makes sense

Reply:
> Yes, they are uniform now 😊

Are you sure? There still seem to be some TBB includes using `<>`, though the majority uses`""`